senior_teacher.rb

```diff
@@ -1,52 +1,19 @@
+require_relative 'dbc_module'
+require_relative 'student'
+require_relative 'teacher_module'
+require_relative 'apprentice_teacher'
+
 class SeniorTeacher
-  attr_reader :age, :salary, :phase, :performance_rating, :target_raise
-  attr_accessor :name
+  include DbcMod
+  include TeacherMod
+  attr_reader :performance_rating
 
   def initialize(options={})
+    super(options)
```

Comment:
@kenrett why did I have to put "super(options)" above/on top of "@phase" to make get senior_teacher spec file to work?